### DISCUSSION TITLE
[Bug]少部分首页推荐视频无法正常打开播放页

### DISCUSSION THREAD

**Discussion by sharkucn** [ywmoyue/biliuwp-lite#624] [bug]
### Bug 描述

点击某些首页推荐视频无法正常打开播放页，但同一个视频从历史记录打开不会出现这个问题（其他设备播放后同步历史记录）

### 操作步骤

```plain text
1. 打开应用
2. 在首页随机点开视频
3. 大多视频可以正常进入播放页，极少部分会报“出现未处理错误”（例如AV112300020072448 / BV1kGJueCEHB）
```

### 期望结果

可以正常从首页播放

### 截图

从首页进入
![image](https://github.com/ywmoyue/biliuwp-lite/assets/135402717/3663b285-479c-40b9-9c7c-c6d089c2314b)
![image](https://github.com/ywmoyue/biliuwp-lite/assets/135402717/5dfe454d-c317-418a-aa1d-16f799063187)

从历史记录进入
![image](https://github.com/ywmoyue/biliuwp-lite/assets/135402717/8626bdde-35fe-4572-8ab0-da854143a034)
![image](https://github.com/ywmoyue/biliuwp-lite/assets/135402717/d2215efe-b268-4b3e-80a9-49dfe5e674cb)

### 日志

[20240511.log](https://github.com/ywmoyue/biliuwp-lite/files/15282369/20240511.log)

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 11 24H2 (26212.5000)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by sharkucn** [comment]
看了下别的视频的AV号，似乎就这个视频长的离谱，不知道是不是和这个有关

**Comment by GD-Slime** [comment]
可复现。通过这个up主的主页打开该视频也会报错。
